pt-BR/mail/chrome/messenger/am-offline.dtd

```diff
@@ -1,10 +1,10 @@
 <!ENTITY doNotDownloadPop3Movemail.label "Para economizar espaço em disco, não receber:">
-<!ENTITY offlineNewFolder.label "When I create new folders, select them for offline use">
+<!ENTITY offlineNewFolder.label "Quando eu criar novas pastas, habilitar o uso offline nelas">
```

Comment:
CORRECTIONS NEEDED it should be +<!ENTITY offlineNewFolder.label "Quando eu criar novas pastas, habilitar que as mesas estejam disponíveis em modo offline">